docker-compose.yml

```diff
@@ -372,12 +368,35 @@ services:
       - "8067:8067"
       - "7070:7070"
       - "8167:8167"
+    expose:
+      - 7000
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "${DEFAULT_LOG_SIZE}"
+        max-file: "${DEFAULT_LOG_FILES}"
+
+
+  #
+  # IRC Services
+  #
+  faf-anope:
+    container_name: faf-anope
+    image: anope/anope
+    restart: unless-stopped
+    networks:
+      - faf
+    depends_on:
+      - faf-ircd
+    volumes:
+      - ./config/faf-anope/services.conf:/anope/conf/services.conf
```

Comment:
I'd always prefer to mount the whole folder if possible

Reply:
That folder contains a dozen of stock configuration files we want to keep upstream, so file is the way